### PR TITLE
Map

### PR DESCRIPTION
**Pull request by savanovic-essex**
- Set up map as background
- Adapted contacts to help user experience in steering the car (according to the new map)
- Set new initial position of the car to fit the new map's start/finish line

### CODE REVIEW

**Review by alros — APPROVED** [review]
💯

File: src/rc_car/car_model/car.py [hunk 1]
```diff
@@ -156,11 +156,11 @@ def _update_acceleration(self, change):
 
     def _steer_right(self, game_time):
         """steer right"""
-        self._steer(game_time, -1)
+        self._steer(game_time, -10)
 
     def _steer_left(self, game_time):
         """steer left"""
-        self._steer(game_time, 1)
+        self._steer(game_time, 10)
```
Comment (marianne-manaog) [inline src/rc_car/car_model/car.py:163]: **Minor suggestion**: the constant `10` could be decoupled to a `constants.py` file and imported from there for ease of maintainability and re-usability.

File: src/rc_car/constants.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 ASSET_CAR = 'car.png'
 ASSET_DRIVER = 'driver.png'
 ASSET_BATTERY = 'battery.png'
+ASSET_BACKGROUND = 'background.png'
```
Comment (marianne-manaog) [inline src/rc_car/constants.py:7]: **Praise**: thanks for adding this - it looks great!

File: src/rc_car/game.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 from car_model.car import Car
 from car_model.remote import Remote
 from constants import (ASSET_BATTERY, ASSET_CAR, ASSET_DIR, ASSET_DRIVER,
-                       BATTERY_HEIGHT, BATTERY_WIDTH, BATTERY_X, BATTERY_Y,
+                       ASSET_BACKGROUND, BATTERY_HEIGHT, BATTERY_WIDTH, BATTERY_X, BATTERY_Y,
```
Comment (marianne-manaog) [inline src/rc_car/game.py:10]: **Minor suggestion**: for next time, please run `isort` for linting and automatically ensuring a correct and consistent ordering of the imported constants (in alphabetical order); i.e., `ASSET_BACKGROUND` should be imported before `ASSET_BATTERY`.

File: src/rc_car/game.py [hunk 4]
```diff
@@ -29,7 +29,7 @@ def __init__(self):
 
         self.driver_image = self._load_image(ASSET_DRIVER, (DRIVER_SIZE, DRIVER_SIZE))
         self.battery_bar = pygame.Surface((BATTERY_WIDTH, BATTERY_HEIGHT))
-        self.car = Car(Vector2(2, 2), self.play_audio, self.check_walls)
+        self.car = Car(Vector2(23, 19), self.play_audio, self.check_walls)
```
Comment (marianne-manaog) [inline src/rc_car/game.py:32]: **Minor suggestion**: the constants `23` and `19` could be decoupled to a `constants.py` file and imported from there for ease of maintainability and re-usability.

**Review by marianne-manaog — COMMENTED** [review]
Hi @savanovic-essex, thanks for your PR! The background looks great!

I have added three comments I would appreciate if you could address in a follow-on PR. 

Cc'ing @alros and @rob-essex

**Pull request merged**